<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_proveedores</class>
 <widget class="QWidget" name="form_proveedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proveedores</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Liberation Sans&quot;;
color: rgb(0, 0, 0);
background-color: rgb(156, 156, 156);
selection-background-color: rgb(164, 190, 221);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>CUIT / CUIL: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_cuit">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Razón Social: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_razon_social">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_buscar">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(119, 149, 177);</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>511</width>
       <height>201</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>501</width>
       <height>201</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget{
background-color: rgb(123, 121, 143);
}
QLineEdit {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QComboBox{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Proveedores / Clientes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Razón Social :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>CUIT / CUIL :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Condición Fte. IVA:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>95</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Referencia: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lne_nombre">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lne_razon_social_proveedor">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lne_cuit_proveedor">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QComboBox" name="cbx_iva">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <item>
              <property name="text">
               <string>Resp. Monotributo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Resp. Inscripto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Exento</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>No Alcanzado</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Consumidor Final</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="lne_ref">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista de Proveedores / Clientes</string>
      </attribute>
      <widget class="QTableWidget" name="tw_proveedores">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>431</width>
         <height>151</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Cuit/Cuil</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Razon Social</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_guardar">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(119, 149, 177);</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_eliminar">
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lne_cuit</tabstop>
  <tabstop>lne_razon_social</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lne_nombre</tabstop>
  <tabstop>lne_razon_social_proveedor</tabstop>
  <tabstop>lne_cuit_proveedor</tabstop>
  <tabstop>cbx_iva</tabstop>
  <tabstop>lne_ref</tabstop>
  <tabstop>btn_guardar</tabstop>
  <tabstop>tw_proveedores</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
